src/test/java/uk/nhs/hee/tis/trainee/sync/service/PlacementSpecialtySyncServiceTest.java

```diff
@@ -60,6 +62,8 @@
 
 class PlacementSpecialtySyncServiceTest {
 
+
+  private static final String _ID = "a001";
   private static final String ID = "40";
   private static final String ID_2 = "140";
   private static final String ID_3 = "2500";
```

Comment:
I know this wasn't you @DorisWongNHS 😁  - but do you think we could do a bit of boy-scouting to make it clearer what IDs are used for what, and to use them a bit more consistently in the tests? Like change 
`_ID` to something like `PLACEMENT_SPECIALTY_ID` 
and `ID` and` ID_2` to `PLACEMENT_ID_1` and `PLACEMENT_ID_2` 
and `ID_3` and `ID_4` to `SPECIALTY_ID_1` and `SPECIALTY_ID_2`